src/oc_erchef/relx.config

```diff
@@ -3,12 +3,7 @@
 %% ex: ts=4 sw=4 ft=erlang et
 {release,{oc_erchef,"1.8.3"},[oc_erchef, syntax_tools, compiler]}.
 {extended_start_script,true}.
-{overlay_vars,"rel/reltool.config"}.
+% Do we need etc/keys? Seems that could live in omnibus?
 {overlay,[{mkdir,"log/sasl"},
           {mkdir,"lib/patches"},
-          {mkdir,"etc/keys"},
-          {copy,"schema","."},
```

Comment:
I think we should move this, but if it causes trouble I can defer this part of the  change for another time.